src/main/treeDataProviders/utils/analyzerUtils.ts

```diff
@@ -0,0 +1,194 @@
+        descriptorNode: DescriptorTreeNode,
+        descriptorData: DescriptorIssuesData
+    ): number {
+        descriptorNode.scannedCve = new Set<string>(descriptorData.applicableIssues?.scannedCve ?? []);
+        descriptorNode.applicableCve = new Map<string, CveApplicableDetails>(
+            descriptorData.applicableIssues ? Object.entries(descriptorData.applicableIssues.applicableCve) : []
+        );
+        descriptorNode.applicableScanTimeStamp = descriptorData.applicableScanTimestamp;
+
+        let issuesCount: number = 0;
+
+        descriptorNode.scannedCve.forEach(cve => {
+            // Check if the descriptor discovered this cve
+            let node: IssueTreeNode | undefined = descriptorNode.getIssueById(cve);
+            if (node instanceof CveTreeNode && node.cve) {
+                let details: CveApplicableDetails | undefined = descriptorNode.applicableCve?.get(node.cve.cve);
+                if (details) {
+                    let evidences: IEvidence[] = [];
+                    // Populate code file issues for workspace
+                    details.fileEvidences.forEach(fileEvidence => {
+                        let fileNode: CodeFileTreeNode = this.getOrCreateCodeFileNode(root, fileEvidence.full_path, node?.severity);
+                        fileEvidence.locations.forEach(location => {
+                            if (location.snippet) {
+                                evidences.push({
+                                    filePathEvidence: this.parseLocationFilePath(fileEvidence.full_path),
+                                    codeEvidence: location.snippet.text
+                                } as IEvidence);
+                                fileNode.issues.push(
+                                    new ApplicableTreeNode(
+                                        cve,
+                                        <CveTreeNode>node,
+                                        fileNode,
+                                        new vscode.Range(
+                                            new vscode.Position(location.startLine, location.startColumn),
+                                            new vscode.Position(location.endLine, location.endColumn)
+                                        ),
+                                        node?.severity
+                                    )
+                                );
+                                issuesCount++;
+                            }
+                        });
+                    });
+                    // Applicable
+                    node.applicableDetails = { isApplicable: true, reason: details.fixReason, evidence: evidences } as IApplicableDetails;
+                } else {
+                    // Not applicable
+                    node.applicableDetails = { isApplicable: false } as IApplicableDetails;
+                }
+            }
+        });
+        return issuesCount;
+    }
+
+    // public static async runEos(
```

Comment:
Let's remove unused code

Reply:
uncommented, this is for the eos scan to keep for future use